### PR TITLE
feat: dismiss validations

### PR DESCRIPTION
**Pull request by rephus**
Closes:  #5649
### Description:

image

### Acceptance criteria

Miro sketch: https://miro.com/app/board/uXjVMNQbbSs=/?moveToWidget=3458764555790284799&cot=14

- [x] I'm able to dismiss items from the validation dashboard by hovering over the item 
- [x] If I dismiss an item, the dismissal is applied for everyone in the app
    - [x] It no longer appears in the validation dashboard
    - [x] It no longer appears in the lists of content with error icons
- [x] You are only able to dismiss an error until it occurs again - this means if I dismiss an error, then the validator is rerun and it occurs again, the error reappears
- [x] Items that are dismissed immediately get removed from the validation dashboard for the user _**and**_ for everyone else in the project.
- [x] when I hover over the dismiss action, I see the helper text `dismiss error`

#### Analytics events:
- [x] When someone dismisses an item, there is an analytics event triggered `validation_error_dismissed`. It has the properties:
    - [x] organization_id
    - [x] project_id
    - [x] user_id - the user who dismissed the error
    - [x] timestamp - when the error was dismissed

#### Documentation

- [x] update the docs to include "how to dismiss errors" section at the bottom of the doc here: https://docs.lightdash.com/guides/validating-your-content 
- [x] explain that if you dismiss an item, it is dismissed for everyone
- [x] explain that if you dismiss an error, but it hasn't been resolved, it will re-appear the next time you run the validator

### CODE REVIEW

**Comment by rephus** [comment]
I forced  a dashboard to be broken for testing the validation on render , that's why some e2e tests are failing.

File: docs/docs/guides/validating-your-content.mdx [hunk 1]
```diff
@@ -36,3 +37,11 @@ The content types included in the validator dashboard are:
 - **Tables** - errors in tables are caused when dimensions or metrics that have been deleted or renamed but are still referenced in other places in the table.
 - **Charts** - errors in charts are caused when the results table or filters reference a metric or dimension which no longer exists.
 - **Dashboards** - errors in dashboards are caused when a chart is broken in the dashboard or a filter references a metric or dimension which no longer exists.
+
+## How to dismiss errors 
+
+You can dismiss errors from validation by clicking on the "x" button that appears after hovering the row on the validation table.
```
Comment (TuringLovesDeathMetal) [inline docs/docs/guides/validating-your-content.mdx:43]:
```suggestion
You can dismiss errors from validation by clicking on the "x" button that appears when you hover over a row on the validation table. 
```

**Review by TuringLovesDeathMetal — APPROVED** [review]
Looks good to me!

I don't know how to test this because there aren't any errors in the demo project 😅 

The one edge case I'd want to test is if the error text is quite long, is there a fixed margin where the text wraps? (basically, we'll never have the `x` overlapping text)

File: packages/frontend/src/components/SettingsValidator/ValidatorTable/index.tsx [hunk 2]
```diff
@@ -44,6 +51,119 @@ const Icon = ({ validationError }: { validationError: ValidationResponse }) => {
...
+                </Anchor>
+            </td>
+            <td>
+                <ErrorMessage validationError={validationError} />
+            </td>
+            <td>
+                <Tooltip label="Dismiss error" position="top">
+                    <Box w={24} h={24}>
+                        {hovered && (
+                            <IconCircleX
```
Comment (IrakliJani) [inline packages/frontend/src/components/SettingsValidator/ValidatorTable/index.tsx:150]: all tabler icons thare are used using `MantineIcon` will get a benefit of passing mantine variables which will not require using a mantine theme hook

`<MantineIcon icon={IconCircleX} color='gray.6' />`

Comment (IrakliJani) [reply packages/frontend/src/components/SettingsValidator/ValidatorTable/index.tsx:150]: also, good alternative to IconCircleX IconCircleXFilled (circle-x-filled) and can be little smaller `<MantineIcon icon={IconCircleX} color='gray.6' size='md' />`

**Review by IrakliJani — COMMENTED** [review]
added minor comments
